setup.py

```diff
@@ -0,0 +1,36 @@
+import sys
+import setuptools
+import pathlib
+
+install_requires = []
+tests_require = ["pytest", "pytest-cov", "pytest-flake8", "asynctest"]
+dev_requires = install_requires + tests_require + ["documenteer[pipelines]"]
+scm_version_template = """# Generated by setuptools_scm
+__all__ = ["__version__"]
+__version__ = "{version}"
+"""
+tools_path = pathlib.PurePosixPath(setuptools.__path__[0])
+base_prefix = pathlib.PurePosixPath(sys.base_prefix)
+data_files_path = tools_path.relative_to(base_prefix).parents[1]
```

Comment:
You can delete the 3 lines above (and probably remove some imports as a result) because you save the config schema in code.